Testing/SystemTests/tests/framework/PolarizationCorrections/HeliumAnalyserEfficiencyTest.py

```diff
@@ -0,0 +1,46 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+from mantid.simpleapi import *
+from PolarizationCorrectionsBase import PolarizationCorrectionsBase
+
+
+class HeliumAnalyserEfficiencyTest(PolarizationCorrectionsBase):
+    def __init__(self):
+        PolarizationCorrectionsBase.__init__(self)
+        self.reference_basename = "HeliumAnalyser"
+        self.input_filename = "ZOOM00038249.nxs"
+
+    def _run_test(self):
+        run = Load(self.input_filename)
```

Comment:
I'd missed that this could be done automatically. Another system test I'd looked at had re-implemented it and so I thought I needed to so too. Thanks for pointing this out!